### ISSUE TITLE
Add Document For Project

### ISSUE THREAD

**Issue by AmBplus** [Maanfee/Maanfee-Dashboard#5]
This Is Realy A Good Blazor Admin Project But I Can't See A Document For Runing And Deploying Project 
I Try To Run Dashboard.Server , But Got Error Can't Connect To Service

**Comment by Maanfee** [comment]
You're right, unfortunately I don't have time to create any document. If you get any error, please tell me so I can guide you.
